### ISSUE TITLE
📃: Adding Beginner Projects 

### ISSUE THREAD

**Issue by TheChaoticor** [UTSAVS26/PyVerse#16]
:red_circle: **Beginner Projects ** :
:red_circle: **To add Beginner Projects ** :
:red_circle: **beginner-friendly Python project ideas to help you practice coding and strengthen your skills.Submit only 1 project and add a text file for description** :

### Screenshots 📷
NA

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Arpit Mishra
- What is your participant role? (Mention the Open Source Program name. Eg. GSOC, GSSOC, SSOC, JWOC, etc.)
   Mentor,gssoc-ext

***********************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by ojhankit** [comment]
@TheChaoticor i would like to get this task assigned to me.

**Comment by UTSAVS26** [comment]
@smartfellow1234 and @ojhankit please raise a new issue telling what you can add in beginner project

**Comment by TheChaoticor** [comment]
Yes,and also both of your projects should be different

**Comment by Charul00** [comment]
can i work on this

**Comment by PasamTharun** [comment]
👋 Hey @UTSAVS26,

I really admire the work you're doing on this project and would love the opportunity to contribute! 🚀 I'm particularly interested in working on Beginner Projects, and I have experience with relevant skills/technologies that I believe can add value.

Could you kindly assign this issue to me? I’m excited to dive in and collaborate with you to make this even better! 😄

Looking forward to hearing from you and getting started!

**Comment by TheChaoticor** [comment]
Yes, see @UTSAVS26 's comment above and raise a new issue mentioning the project 

> can i work on this

**Comment by Pavitra-s-Workspace** [comment]
Uppada Pavitra Manaswini Reddy
GSSoC24' Extended Contributor
Im interested to contribute please assign me.

**Comment by UTSAVS26** [comment]
@Charul00 @Pavitra-s-Workspace @PasamTharun please raise a new issue telling what you can add in beginner project. Allt the best

**Comment by TheChaoticor** [comment]
Everyone, please raise an issue as fast as you can, as we have to assign people.Max assignment on this one is 10

**Comment by TheChaoticor** [comment]
Everyone, please raise an issue as fast as you can, as we have to assign people.

**Comment by Yash-Bhatnagar-02** [comment]
Yash Bhatnagar
Hacktoberfest Contributor
Please assign this issue to me. Attached is my project of quering in multiple PDFs:
https://github.com/Yash-Bhatnagar-02/Chat-with-multiple-PDFs.git
